applications/DEM_application/custom_elements/rigid_body_element.cpp

```diff
@@ -200,7 +200,7 @@ namespace Kratos {
         center_forces[0] = center_forces[1] = center_forces[2] = center_torque[0] = center_torque[1] = center_torque[2] = 0.0;
 
         #pragma omp parallel for schedule(dynamic, 100)
-        for (unsigned int k = 0; k < mListOfNodes.size(); k++) {
+        for (int k = 0; k < (int)mListOfNodes.size(); k++) {
```

Comment:
I would suggest the use of static_cast for casting:

```c++
for (int k = 0; k < static_cast<int>(mListOfNodes.size()); k++) {
```
The reasoning is the fact that now the casting is more visible and is obvious that a compile time casting is done.